wrappers/java/wrapper

```diff
@@ -33,23 +33,27 @@ wrapper_dir=$(dirname "$0")
 cd "$wrapper_dir"
 
 if [ "$action" == "version" ]; then
-	exec ant -S -q version
+	exec mvn --version
 elif [ "$action" == "compile" ]; then
 	if [[ -z "$module" ]]; then
 		echo "Syntax error"
 		echo "$usage"
 		exit 1
 	fi
-	#echo "compiling:" ant "-Dorg.ensembl.hive.module=$module" compile_wrapper
-	exec ant "-Dorg.ensembl.hive.module=$module" compile_wrapper
+	#echo "compiling:" mvn package
+	mvn package
+  	exec java -cp target/eHive-*-jar-with-dependencies.jar org.ensembl.hive.CompileWrapper "$module"
```

Comment:
Indentation should be tabulation only in this file. This line starts with some whitespace